graphics/GLC_lib/Portfile

```diff
@@ -38,6 +38,8 @@ configure.args      GLC_lib.pro
 # https://github.com/laumaya/GLC_lib/issues/30
 compiler.cxx_standard 2003
 configure.cxxflags-append -std=c++03
+# Setting 2003 standard does not prevent Xcode gcc-4.* from being picked, but they do not acknowledge -std=c++03 flag.
+compiler.blacklist-append *gcc-4.0 *gcc-4.1 *gcc-4.2
```

Comment:
The `-std=c++03` flag was first recognized by GCC 4.7, although it appears to be equivalent to `-std=c++98`: https://github.com/gcc-mirror/gcc/commit/c7dc8804a86017317cd7d3e5f65f715d5c4d5da2

Is this build error merely due to the flag not being recognized, such that using `-std=c++98` instead would work? Or does this port need C++03 behavior only available in GCC ≥ 4.3?

If the latter, then ideally that can be handled in MacPorts base (which currently treats `compiler.cxx_standard 2003` the same as `compiler.cxx_standard 1998`) and mentioned on https://trac.macports.org/wiki/CompilerSelection

Reply:
@chrstphrchvz Thank you, I will try that.

Reply:
UPD. Yeah, seems to work fine.